interface/super/rules/controllers/edit/helper/common.php

```diff
@@ -107,7 +114,7 @@
 <!--                  -->
 <?php function timeunit_select($args)
 {
-    require_once($GLOBALS["srcdir"] . "/options.inc.php");
+    include_once $GLOBALS["srcdir"] . "/options.inc.php";
```

Comment:
keep this a require_once

require_once will cause a fatal error if the file is missing, which is what we want.
(include_once will only throw a php warning if the file is missing)

Reply:
@bradymiller, hmm. Very strange. I don't recall changing require_once to include_once. I did run phpcbf on the file and it caused changes to the left and right parenthesis in these calls. But it should not have modified the function name. I will make sure require_once is used. Thanks

Reply:
hi @hanksterr7, pretty sure it's not phpcbf but it might be a setting in your IDE like phpstorm for example.